lusidtools/jupyter_tools/notebook_stopper.py

```diff
@@ -0,0 +1,12 @@
+class StopExecution(Exception):
+    """
+    This class is used to stop a notebook from running.
+    A sample implementation - stop the notebook if a certain portfolio does not exist
```

Comment:
I would remove this line and give an example of how to use it e.g. `raise StopExecution("missing portfolio")`

Reply:
Done.
See

Reply:
https://github.com/finbourne/lusid-python-tools/pull/131